.github/workflows/test_app.yml

```diff
@@ -92,7 +91,9 @@ jobs:
           path: ./spec/decidim_dummy_app/
           key: app-${{ github.sha }}
           restore-keys: app-${{ github.sha }}
-      - run: bundle exec rake parallel:create parallel:load_schema
+      - run: |
+          bundle install
```

Comment:
There is a problem with the way resources are being cached by GitHub actions, and the 7zip.so file is not included in the cache digest. Without this `bundle install`, we cannot force the generation of the needed 7zip.so.

Reply:
Thanks for the info!